main/test/ca/uwaterloo/flix/library/TestArray.flix

```diff
@@ -13,6 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Remember to add your name to the copyright at the top. (May apply to other files as well).